upgrade/satellite.py

```diff
@@ -120,6 +120,14 @@ def satellite6_upgrade(zstream=False):
     disable_repo_name = ["*"]
     enable_repos_name = ['rhel-{0}-server-rpms'.format(major_ver),
                          'rhel-server-rhscl-{0}-rpms'.format(major_ver)]
+
+    # This statement will execute only until downstream release not become beta.
+    if os.environ.get('NON_BETA_FM_UPGRADE') == 'true':
+        enable_disable_repo(disable_repo_name, enable_repos_name)
```

Comment:
Is it repeating below?

Reply:
Yes, that is for a non-fm upgrade, I handled in the above fix.

Reply:
Fixed

Reply:
I think FM automatically take cares of repo enablement but not for Downstream builds.